common/scripts/configs/master_integration_fields.sql

```diff
@@ -934,6 +934,21 @@ do $$
       values (245, '59eee00ef7bcf03ff7b62fc7', 'webhookUrl', 'string', true, false,'54188262bc4d591ba438d62a', '54188262bc4d591ba438d62a', '2017-10-24', '2017-10-24');
     end if;
 
+    -- masterIntegrationFields for nodeCluster
+    if not exists (select 1 from "masterIntegrationFields" where "id" = 246) then
+      insert into "masterIntegrationFields" ("id", "masterIntegrationId", "name", "dataType", "isRequired", "isSecure","createdBy", "updatedBy", "createdAt", "updatedAt")
+      values (246, '59eee00ef7bcf03ff7b62fc8', 'nodes', 'array', true, false,'54188262bc4d591ba438d62a', '54188262bc4d591ba438d62a', '2016-06-01', '2016-06-01');
```

Comment:
- Space after `false,` here and in the next few fields. 
- Let's also add today's date for `createdAt` and `updatedAt`. 

These were missed in the new integrations that we added. We can clean them up later, but lets create the new ones correctly.